_build/templates/default/sass/_forms.scss

```diff
@@ -1548,3 +1551,12 @@ td.x-date-mp-sep {
     background-color: rgba(0, 0, 0, .8);
   }
 }
+
+// Template preview field that opens the file browser
+#x-form-el-modx-template-preview-file {
```

Comment:
This line can be moved higher, and use variables, as in https://github.com/modxcms/revolution/blob/3.x/_build/templates/default/sass/_forms.scss#L479

Reply:
Resolved this by adding a triggerClass in the JS instead of adding new CSS. Thanks!